src/adr/0028-Recompute-mailbox-quotas.md

```diff
@@ -0,0 +1,47 @@
+# 28. Recompute mailbox quotas
+
+Date: 2020-04-03
+
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+JMAP custom quota extension, as well as IMAP [RFC-2087](https://tools.ietf.org/html/rfc2087) enables a user to monitor
+the amount of space and message count he is allowed to use, and that he is effectively using.
+
+To track the quota values a user is effectively using James relies on the 
```

Comment:
Something is wrong in this sentence... I think I would drop the `is`

Reply:
More a coma